From bdd649f1c7bd964386f878ecbaf8ae5bb92503cf Mon Sep 17 00:00:00 2001
From: Sean Farrell <sean.farrell@rioki.org>
Date: Sun, 19 Feb 2023 16:02:36 +0100
Subject: [PATCH] Remove find_package(gumbo)

The gumbo package is actually never installed, just statically linked to.
---
 cmake/litehtmlConfig.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmake/litehtmlConfig.cmake b/cmake/litehtmlConfig.cmake
index 5eedcf4..f90bdf8 100644
--- a/cmake/litehtmlConfig.cmake
+++ b/cmake/litehtmlConfig.cmake
@@ -1,3 +1,2 @@
 include(CMakeFindDependencyMacro)
-find_dependency(gumbo)
 include(${CMAKE_CURRENT_LIST_DIR}/litehtmlTargets.cmake)
-- 
2.33.0.windows.2

